neurovault/apps/statmaps/templates/statmaps/search.html

```diff
@@ -70,8 +70,8 @@
 
 		</script>
 		<!-- <gcse:search linkTarget="_self"></gcse:search> -->
-    <gcse:searchbox-only resultsUrl="http://192.168.99.103/search" queryParameterName="as_q"></gcse:searchbox-only>
-    <gcse:searchresults-only queryParameterName="as_q" linkTarget="_self"></gcse:searchresults-only>
+    <gcse:searchbox-only resultsUrl="http://neurovault.org/search"></gcse:searchbox-only>
```

Comment:
I don't think so. There is an other way to do this but it is indirect. This is the best way possible from the documentation of CSE. :) I will try looking for it.